src/components/Accordion/Accordion.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file Accordion.
+ * @copyright IBM Security 2019
+ */
+
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import CarbonAccordion from 'carbon-components-react/lib/components/Accordion';
+
+import { getComponentNamespace } from '../../globals/namespace';
+
+const namespace = getComponentNamespace('accordion');
+
+const { defaultProps, propTypes } = CarbonAccordion;
+
+const Accordion = ({ className, alignment, ...rest }) => {
```

Comment:
- Can we call this prop `align` and use the values `start` and `end` to maintain consistency with Carbon? https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/TooltipDefinition/TooltipDefinition.js#L78
- For consistency with the majority of our spread props, can we use `...other` here?